tests/plugins/widget/integration/selection/manual/ieinputwidget.html

```diff
@@ -0,0 +1,43 @@
+<div id="editor">
+		<p>Add widget below.</p>
+	</div>
+
+	<script>
+		if ( !CKEDITOR.env.ie || CKEDITOR.env.version < 11) {
```

Comment:
The changed code path touches all IE browsers (including Edge). Is there a reason why this test is enabled only for IE11 and Edge?

Reply:
Those are 2 main branches of browser which we actively support, so I haven't test it on lower IE versions.
However, I've checked and everything seems to work up to IE9 with manual and unit tests. IE8 has some troubles with this fix. It doesn't sound reasonable to spend too much time on preparing dedicated for IE8 anyway. I think we can remain it as it is, and if there appear requests for supporting such cases under IE8, then fix it.